app/models/piece.rb

```diff
@@ -61,15 +61,14 @@ def players_own_piece_is_there?(x, y)
 
   def move_to!(x, y)
     occupying_piece = Piece.where(x_position: x, y_position: y, game_id: game.id)
+    occupying_piece.first&.set_captured!
+
     last_piece_moved = game.pieces.order('updated_at').last
-    if occupying_piece.any? then
-      occupying_piece.first.set_captured!
-      occupying_piece.first.save
-    end
     if en_passant?(x, y) then
       last_piece_moved.set_captured!
       last_piece_moved.save
```

Comment:
Not a huge deal at all for this PR but in the back of our heads we should remember if we do a clean up code pass that since `.set_captured!` now saves we should not need this line.

Reply:
You're talking about the `save` line right? The code above is showing what was removed.

Reply:
Yes it was line 71. It is now line 69.